geomesa-plugin/src/main/scala/org/locationtech/geomesa/plugin/wms/CoverageReader.scala

```diff
@@ -71,11 +72,11 @@ class CoverageReader(val url: String) extends AbstractGridCoverage2DReader() wit
     case (columnFamily, columnQualifier) => new Text("~" + columnFamily + "~" + columnQualifier)
   }
 
-  this.crs = AbstractGridFormat.getDefaultCRS
-  this.originalEnvelope = new GeneralEnvelope(Array(-180.0, -90.0), Array(180.0, 90.0))
-  this.originalEnvelope.setCoordinateReferenceSystem(this.crs)
-  this.originalGridRange = new GridEnvelope2D(new Rectangle(0, 0, 1024, 512))
-  this.coverageFactory = CoverageFactoryFinder.getGridCoverageFactory(this.hints)
+  crs = CRS.decode("EPSG:4326")
```

Comment:
I did a similar thing with the raster branch in another PR, I was advised to use a Try around this with the else being DefaultGeographicCRS.WGS84.